loan-payoff/LETTERSPECS/BANNO.LOAN.PAYOFF.V1.CFG

```diff
@@ -1,8 +1,36 @@
+************************************************************************
+* BANNO.LOANPAYMENT.SKIP.CFG
+*
+* Copyright 2020-2024 Jack Henry & Associates, Inc.
+*
 * This letterfile contains parameters for the BANNO.LOAN.PAYOFF.V1.POW
 * specfile. Each line starting with an asterisk will be considered a
 * comment so feel free to add additional lines as necessary.  Each
 * line in this letterfile cannot be longer than 126 characters.
 *
+************************************************************************
+************************************************************************
+* Modification History:
+*  Ver. 1.0.0  01/27/2020: C. Simms - Original Programming
```

Comment:
I pulled it directly from the PowerOn's Comments at the top, but needed a little rewording in some places so it would make sense in the CFG. Only included the program creation date, any comments that said a parameter was added, and then included the new parameter, PID, of Tom's I'm adding:
BANNO.LOAN.PAYOFF.V1.POW
...
**  Modification History:
****    Ver. 1.0.0  01/27/20 C. Simms - Original Programming**
...
****    Ver. 1.1.0  08/04/22 R. Robison - New features
**                         Added support for payoff fee display in the UX.**
**    Ver. 1.1.1  06/18/24 J. Keenan - Updated the loggingErrorMessage
**                         for ineligible loans and ineligible
**                         warning codes to include the loan type and
**                         loan warning code numbers.
**                         Also, added Program Installation Date parameter setting